kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionEditor.vue

```diff
@@ -557,6 +561,7 @@
     left: 0;
     padding: 1em;
     margin-top: 1em;
+    background-color: #ffffff;
```

Comment:
Should we be setting this to themeTokens.surface instead? https://github.com/learningequality/kolibri-design-system/blob/release-v4/lib/styles/colorsDefault.js#L18